vxquery-core/src/test/java/org/apache/vxquery/datamodel/AbstractPointableTest.java

```diff
@@ -28,7 +28,9 @@
 
     protected void getTaggedValuePointable(Object value, IPointable result) throws IOException {
```

Comment:
See my other comments. Object build should not remove the tag, but require a UTF8StringPointable as the argument. Let the calling class ensure a UTF8StringPointable as the key argument.